package-lock.json

```diff
@@ -1563,8 +1563,7 @@
          "version": "1.2.0",
          "resolved": "https://registry.npmjs.org/aproba/-/aproba-1.2.0.tgz",
          "integrity": "sha512-Y9J6ZjXtoYh8RnXVCMOU/ttDmk1aBjunq9vO0ta5x85WDQiQfUF9sIPBITdbiiIVcBo03Hi3jMxigBtsddlXRw==",
-         "dev": true,
-         "optional": true
+         "dev": true
```

Comment:
Is this change due to a different npm version?

But anyway, as you're not changing `package.json`, you can just remove `package-lock.json` from this commit.

Reply:
Not sure, but removed the file. Thank you!